src/test/java/com/google/api/codegen/protoannotations/testdata/java_library_no_gapic_config.baseline

```diff
@@ -1695,7 +1832,386 @@ public class LibraryServiceClient implements BackgroundResource {
     return stub.awaitTermination(duration, unit);
   }
 
+  public static class ListShelvesPagedResponse extends AbstractPagedListResponse<
```

Comment:
This new class is also present in java_library.baseline: https://github.com/googleapis/gapic-generator/blob/eea42da690a5cec5660a76c540a45465323128af/src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline#L3630